boards/arm/olimexino_stm32/Kconfig.defconfig

```diff
@@ -44,4 +44,20 @@ config PWM_STM32_1
 
 endif # PWM
 
+if USB
+
+config USB_DC_STM32_DISCONN_ENABLE
```

Comment:
Can we make this a 'select USB_DC_STM32_DISCONN_ENABLE 
 if USB' boards/arm/olimexino_stm32/Kconfig.board

Reply:
Done.